lms/djangoapps/teams/views.py

```diff
@@ -111,6 +111,7 @@ def get(self, request, course_id):
             "topics_url": reverse('topics_list', request=request),
             "teams_url": reverse('teams_list', request=request),
             "team_memberships_url": reverse('team_membership_list', request=request),
+            "teams_membership_detail_url": reverse('team_membership_detail', args=['team_id', user.username]),
```

Comment:
Can we make it just `team_membership_detail_url ` to keep the pluralization consistent?